backend/src/main/java/com/soen341/instagram/model/Likable.java

```diff
@@ -0,0 +1,29 @@
+package com.soen341.instagram.model;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.persistence.ManyToMany;
+import javax.persistence.MappedSuperclass;
+
+@MappedSuperclass
+public abstract class Likable {
+	@ManyToMany
+	private Set<Account> likedBy;
+	
+	public Likable() {
+		likedBy = new HashSet<Account>();
+	}
+	
+	public Set<Account> getLikedBy() {
+		if (likedBy.equals(null))
```

Comment:
this line is incorrect because `likedBy.equals` will error if `likedBy` is null
this should be `if (likedBy == null)`